modules/gitops-release-notes-1-5-0.adoc

```diff
@@ -0,0 +1,63 @@
+** Supports Argo CD RBAC using OpenShift groups.
+** Honors `HTTP_Proxy`, RH-SSO can be used as an SSO for Argo CD running behind a proxy. 
++
+link:https://issues.redhat.com/browse/GITOPS-1330[GITOPS-1330]
+
+* With this update, a new URL field `.host` is added to the `.status` of the Argo CD. When route or ingress is enabled (priority given to route) then the route will be displayed in the new URL field. If no URL is provided from the route or ingress, the `.host` field will not be displayed.
++
+When the route or ingress is configured, but the corresponding controller is not set up properly (i.e. is not in Ready state or does not propagate its URL), this is indicated in the Operand as well in the value of `.status.host` as `Pending` instead of the URL. This affects the overall status for the Operand by making it `Pending` instead of `Available`. link:https://issues.redhat.com/browse/GITOPS-654[GITOPS-654]
+
+[id="fixed-issues-1-5-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, RBAC rules specific to *AppProjects* would not allow the use of commas for the role's subject field, thus preventing bindings to LDAP account. This has been fixed and complex role bindings can now be specified in *AppProject* specific RBAC rules. link:https://issues.redhat.com/browse/GITOPS-1771[GITOPS-1771]
+
+* Before this update, when a `DeploymentConfig` is scaled to `0`, ArgoCD used to display it in *progressing* state with health status message as *"replication controller is waiting for pods to run"*. With this update, the edge case is fixed and the health check will report the correct health status of `DeploymentConfig` resource. link:https://issues.redhat.com/browse/GITOPS-1738[GITOPS-1738]
+
+* Before this update, the TLS certificate in the `argocd-tls-certs-cm` configuration map was deleted by the {gitops-title} unless the certificate was configured in the ArgoCD CR specification `tls.initialCerts` field. This update fixes this issue. link:https://issues.redhat.com/browse/GITOPS-1725[GITOPS-1725]
+
+* Before this update, while creating a namespace with the `managed-by` label it created a lot of `RoleBinding` on the new namespace. This update fixes the issue of {gitops-title} removes the extraneous `Role` and `RoleBinding` resources that were created by the previous versions.
+ link:https://issues.redhat.com/browse/GITOPS-1550[GITOPS-1550]
+
+* Before the update,  Firefox 94 and later failed to connect to Argo CD UI with error code *SEC_ERROR_BAD_DER* since TLS certificate of the Route in pass-through mode did not not had a CA name. link:https://issues.redhat.com/browse/GITOPS-1548[GITOPS-1548]
++
+With this update, first delete the `<openshift-gitops-ca>` secrets, let it recreate and then delete `<openshift-gitops-tls>`. After recreating again by the {gitops-title}, the Argo CD UI is accessible by Firefox. link:https://issues.redhat.com/browse/GITOPS-1548[GITOPS-1548]
```

Comment:
Suggestion: Move it up and align it with the previous paragraph.

Reply:
both are getting created by the Operator. SMEs already checked and based on their suggestion the content was changed